packages/server/src/errorResponse.json

```diff
@@ -21,6 +21,9 @@
   "api": {
     "authentication": {
       "invalidEmail": "Enter valid email",
+      "invalidEmailFormat": "Invalid email format",
+      "invalidEmailDomain": "Invalid email format",
```

Comment:
@Vedant005 both of the keys has same value.

Reply:
@Vedant005 Once you've updated the code for related review, it's best practice to share an quick update in the comment or mark the review as "Resolve conversation". 🙂